app/ui-react/syndesis/scripts/minishift-restore.sh

```diff
@@ -83,3 +83,6 @@ oc replace --force -f - <<EOF
   ]
 }
 EOF
+
+# also scale back up the operator
+oc scale --replicas=1 dc syndesis-operator
```

Comment:
@zregvart is that `dc` bit there needed? @igarashitm made me notice it, maybe it's not needed?